man/check_gambl_metadata.Rd

```diff
@@ -19,6 +19,9 @@ check_gambl_metadata(
 \item{show_details}{Set to TRUE if you want the gory details about issues that are identified}
 
 \item{fix}{After identifying an issue, rerun this function with fix=TRUE to address errors (when possible). Currently this doesn't do anything. That's how I roll}
+}
```

Comment:
A lot of .Rd's have been updated with these lines, could you educate me to why this is?

Reply:
They were added when I run the `devtools::document()`, I think they were just missing or did not reflect the updated arguments